src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala

```diff
@@ -33,7 +39,10 @@ import scala.util.{ Failure, Success, Try }
  * @param submitToBaseDir the directory to which the staged copy must be moved.
  */
 class Submitter(stagingBaseDir: File,
-                submitToBaseDir: File) {
+                submitToBaseDir: File,
+                group: String,
+               ) {
+  private val groupPrincipal = stagingBaseDir.path.getFileSystem.getUserPrincipalLookupService.lookupPrincipalByGroupName(group)
```

Comment:
Making this variable lazy would postpone the error from service start to the first request, but is that desired behaviour?

Reply:
I think we should fail with this ASAP, preferably on startup! However, isn't this already the case with the `lazy val` for the `Submitter itself?

Perhaps you should do something similar to `getConfiguredDirectory` with this `groupPrinciple` in `EasyDepositApiApp` and give the `GroupPrinciple` as an argument to `Submitter`.